tests/unit/FilingHistoryList2.spec.ts

```diff
@@ -145,6 +150,44 @@ filings.forEach((filing: any, index: number) => {
       expect(wrapper.find('.hide-details').text()).toBe('Hide Documents')
     })
 
+    itIf(isHorizontalAmalgamation(filing))('horizontal amalgamation filing', async () => {
+      // verify data
+      expect(vm.getFilings.length).toBe(1) // sanity check
+      const item = vm.getFilings[0]
+      expect(item.businessIdentifier).toBe('BC1234567')
+      expect(item.data.amalgamationApplication.type).toBe(FilingSubTypes.AMALGAMATION_HORIZONTAL)
+      expect(item.displayName).toBe('Amalgamation Application - Horizontal')
+      expect(item.name).toBe(FilingTypes.AMALGAMATION_APPLICATION)
+      expect(item.status).toBe(FilingStatus.COMPLETED)
+
+      // verify display
+      expect(wrapper.find('.item-header-title').text()).toBe('Amalgamation Application - Horizontal')
+      expect(wrapper.find('.item-header-subtitle').text()).toContain('FILED AND PAID')
+      expect(wrapper.find('.item-header-subtitle').text()).toContain('(filed by Registry Staff on Feb 3, 2023)')
+      expect(wrapper.find('.item-header-subtitle').text()).toContain('EFFECTIVE as of Feb 3, 2023')
+      expect(wrapper.find('.view-details').text()).toBe('View Documents')
+      expect(wrapper.find('.hide-details').text()).toBe('Hide Documents')
+    })
+
+    itIf(isVerticalAmalgamation(filing))('vertical amalgamation filing', () => {
+      // verify data
+      expect(vm.getFilings.length).toBe(1) // sanity check
+      const item = vm.getFilings[0]
+      expect(item.businessIdentifier).toBe('BC1234567')
+      expect(item.data.amalgamationApplication.type).toBe(FilingSubTypes.AMALGAMATION_VERTICAL)
+      expect(item.displayName).toBe('Amalgamation Application - Vertical')
+      expect(item.name).toBe(FilingTypes.AMALGAMATION_APPLICATION)
+      expect(item.status).toBe(FilingStatus.COMPLETED)
+
+      // verify display
+      expect(wrapper.find('.item-header-title').text()).toBe('Amalgamation Application - Vertical')
+      expect(wrapper.find('.item-header-subtitle').text()).toContain('FILED AND PAID')
+      expect(wrapper.find('.item-header-subtitle').text()).toContain('(filed by Registry Staff on Feb 3, 2023)')
+      expect(wrapper.find('.item-header-subtitle').text()).toContain('EFFECTIVE as of Feb 3, 2023')
+      expect(wrapper.find('.view-details').text()).toBe('View Documents')
+      expect(wrapper.find('.hide-details').text()).toBe('Hide Documents')
+    })
```

Comment:
Very nice! This goes along with updating the sample filings response (filings.json below).

Now you know everything :)